practice_05.25.19_interface/Task2_BankAccount/Bank.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace practice_05._25._19_interface.Task2_BankAccount
+{
+    public class Bank
+    {
+        private readonly List<BankAccount> accounts;
+
+        public Bank()
+        {
+            accounts = new List<BankAccount>();
+        }
+
+        public List<BankAccount> ManageAccount()
+        {
+            while (true)
+            {
+                Console.WriteLine("Do you want to add new account? Y/N");
+                string action = Console.ReadLine();
+                if (action.ToLower() == "y")
+                {
+                    while (true)
+                    {
+                        Console.WriteLine("Choose one of the account that you want to add:\n" +
+                            "1 - Bank Account\n" +
+                            "2 - Deposit Account\n" +
+                            "3 - Card Account\n" +
+                            "4 - Current Account\n" +
+                            "0 - to return in previous menu");
+                        string accountType = Console.ReadLine();
+                        if (accountType == "1")
```

Comment:
Please use switch statement in such cases.

Reply:
fixed - switch statement was added instead of 'else if'